src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -205,11 +207,18 @@ export function queryForFacilities(input = '') {
 
 export const disabilityIsSelected = disability => disability['view:selected'];
 
+/**
+ * Takes a string and returns another that won't break SiP when used
+ * as a property name.
+ * @param {string} str - The string to make SiP-friendly
+ * @return {string} The SiP-friendly string
+ */
+export const sippableId = str => (str || 'blank').toLowerCase();
```

Comment:
I originally didn't have the `|| 'blank'`, but it turns out It's called when `disability.condition === undefined`, so we were running into a `cannot call toLowerString of undefined` issue. I use `'blank'` rather than `''` because I don't think we can actually use the latter as an object key (to be fair, I didn't test that assumption).